cmd/tealdbg/localLedger.go

```diff
@@ -17,17 +17,33 @@
 package main
 
 import (
+	"encoding/json"
 	"fmt"
+	v2 "github.com/algorand/go-algorand/daemon/algod/api/server/v2"
+	"io/ioutil"
 	"math/rand"
+	"net/http"
 
 	"github.com/algorand/go-algorand/config"
 	"github.com/algorand/go-algorand/crypto"
+	"github.com/algorand/go-algorand/daemon/algod/api/server/v2/generated"
 	"github.com/algorand/go-algorand/data/basics"
 	"github.com/algorand/go-algorand/data/transactions"
 	"github.com/algorand/go-algorand/data/transactions/logic"
 	"github.com/algorand/go-algorand/ledger"
 )
 
+type AccountResponse struct {
```

Comment:
suggest AccountIndexerResponse name to make it clear - AccountResponse is a thing for algod response